python/lsst/ts/salobj/base_csc.py

```diff
@@ -190,7 +190,12 @@ def make_from_cmd_line(cls, index, **kwargs):
         """
         parser = argparse.ArgumentParser(f"Run {cls.__name__}")
         if index is True:
-            parser.add_argument("index", type=int, help="Script SAL Component index.")
+            parser.add_argument("index", type=int, help="SAL index.")
+        parser.add_argument(
+            "--loglevel",
+            type=int,
+            help="log level: error=40, warning=30, info=20, debug=10",
```

Comment:
Consider using `choices = [40, 30, 20, 10]` or maybe even naming the levels and parsing them afterwards. Unless the intention is to support non standard levels.

Reply:
The intent is to support all levels.